### PR TITLE
Add appeal reason code seed and include it in `default.vw_pin_appeal`

### PR DESCRIPTION
**Pull request by wrridgeway**

### CODE REVIEW

**Comment by wrridgeway** [comment]
dbt build is currently failing because there are bad values for `change` in `iasworld.htpar`. Mirella has already fixed them and dbt should build without issue tomorrow 🤞 .

File: dbt/seeds/ccao/docs.md [hunk 1]
```diff
@@ -35,6 +35,15 @@ To find the level of assessment (LoA) for each class, see the `ccao.loa` table.
 **Primary Key**: `class_code`
 {% enddocs %}
 
+# htpar_reascd
+
+{% docs seed_htpar_reascd %}
+Table containing descriptions for appeal decision reason codes from
+`iasworld.htpar`.
```
[the remark this answers is not in the document]
Comment (wrridgeway) [reply dbt/seeds/ccao/docs.md:42]: Good call. They've been living as an attachment on the open data portal forever now, my guess is we got them from Mirella/Will. I'll look through my email and add a note.

[the remark this answers is not in the document]
Comment (ccao-jardine) [reply dbt/seeds/ccao/docs.md:42]: I can't speak to the source of the tabular version, just commenting to note that they are public as a [pdf doc](https://www.cookcountyassessor.com/form-document/assessor-reason-codes) on the CCAO website.

[the remark this answers is not in the document]
Comment (jeancochrane) [reply dbt/seeds/ccao/docs.md:42]: I like the idea of linking to the PDF doc!

[the remark this answers is not in the document]
Comment (wrridgeway) [reply dbt/seeds/ccao/docs.md]: Sure thing. There are definitely some codes in iasWorld that aren't in the PDF, but I'll include it as a resource regardless.

**Comment by wrridgeway** [comment]
@jeancochrane Would it make any sense to move `default_vw_pin_appeal_no_unexpected_change_values` into the qc views? It seems weird to have the view [fail to build](https://github.com/ccao-data/data-architecture/actions/runs/11445926440/job/31844013959) if someone enters bad data in iasWorld.

**Comment by jeancochrane** [comment]
> Would it make any sense to move `default_vw_pin_appeal_no_unexpected_change_values` into the qc views? It seems weird to have the view [fail to build](https://github.com/ccao-data/data-architecture/actions/runs/11445926440/job/31844013959) if someone enters bad data in iasWorld.

@wrridgeway I agree that we need to rethink this test, but I'm not totally sure what the best path forward is. Check out the distribution of results for this query:

```sql
with user104s as (
    select TRIM(LOWER(user104)) as user104
    from iasworld.htpar
)

select user104, count(*) from user104s group by user104
```

It seems like the underlying data in `htpar` no longer match the assumptions of our view. Its values appear to be only `"change"`, `"no change"`, and null (discounting the two problematic rows that caused the test to start erroring out). When I add `taxyr` to the `user104s` query and group by that in the top-level `select` statement, I also see that all years prior to 2020 appear to be null across the board, with no values at all let alone the legacy `"C"`, `"NC"`, and `"decrease"` values.

If the distribution in `htpar.user104` is correct and not the result of some sort of systemic data problem, I wonder if the best path forward for us would be to adjust the definition of the `vw_pin_appeal.change` field so that it selects `TRIM(LOWER(user104))` for all years, and move the `default_vw_pin_appeal_no_unexpected_change_values` test to the `iasworld.htpar` source definition?

**Comment by wrridgeway** [comment]
@jeancochrane I think we can clean up the code that constructs `change` (remove 'decrease' clause), but i think we're already anticipating `user104` being empty before 2021 - we use [`resact`](https://github.com/ccao-data/data-architecture/blob/9ce36925ef9e42814826ef8d4287835c749c97b6/dbt/models/default/default.vw_pin_appeal.sql#L75) for 2020 and prior.

**Pull request merged**